pkg/k8s/factory_functions.go

```diff
@@ -926,3 +926,27 @@ func ObjToCiliumEndpointBatch(obj interface{}) *cilium_v2alpha1.CiliumEndpointBa
 		Warn("Ignoring invalid CiliumEndpointBatch")
 	return nil
 }
+
+// convertCepToCoreCep converts a CiliumEndpoint to a CoreCiliumEndpoint
+// containing only a minimal set of entities used to
+func ConvertCepToCoreCep(cep *cilium_v2.CiliumEndpoint) *cilium_v2alpha1.CoreCiliumEndpoint {
+
+	// Copy Networking field into core CEP
+	var epNetworking *cilium_v2.EndpointNetworking
+	if cep.Status.Networking != nil {
+		epNetworking = new(cilium_v2.EndpointNetworking)
+		cep.Status.Networking.DeepCopyInto(epNetworking)
+	}
+	var identityID int64 = 0
```

Comment:
type is incorrect, see https://github.com/cilium/cilium/blob/1a7a8ad4a89fd9bbe3ee7e6bb3cd7fdaaa251f58/pkg/identity/numericidentity.go#L351

Reply:
`EndpointIdentity` ID is `int64` type.
https://github.com/cilium/cilium/blob/master/pkg/k8s/apis/cilium.io/v2/types.go#L209-L215

Reply:
Ah you are right, interesting, I wonder why these two don't match?